pages/bugs.jsx

```diff
@@ -42,27 +46,95 @@ export default function BugsPage() {
     setShowDonated(!showDonated);
   }
 
+  const sortName = () => {
+    setCrittersToShow(
+      [...crittersToShow].sort((critterA, critterB) => {
+        const nameA = critterA.name.toUpperCase();
+        const nameB = critterB.name.toUpperCase();
+        if (isAsc) {
+          if (nameA < nameB) {
+            return -1;
+          }
+          if (nameA > nameB) {
+            return 1;
+          }
+        } else {
+          if (nameA > nameB) {
+            return -1;
+          }
+          if (nameA < nameB) {
+            return 1;
+          }
+        }
+      })
+    );
+    setIsAsc(!isAsc);
+  };
+
+  const sortPrice = () => {
+    setCrittersToShow(
+      [...crittersToShow].sort((critterA, critterB) => {
+        const priceA = parseInt(critterA.price);
+        const priceB = parseInt(critterB.price);
+        if (isAsc) {
+          if (priceA < priceB) {
+            return -1;
+          }
+          if (priceA > priceB) {
+            return 1;
+          }
+        } else {
+          if (priceA > priceB) {
+            return -1;
+          }
+          if (priceA < priceB) {
+            return 1;
+          }
+        }
+      })
```

Comment:
```suggestion
      [...crittersToShow].sort((critterA, critterB) => parseInt(critterA.price) - parseInt(critterB.price) * isAsc ? 1 : -1)
```
For numbers you don't need to return 1 / -1, you could also just return ```a - b``` for ascending or ```b - a``` for descending.
![image](https://user-images.githubusercontent.com/15978350/173956185-f192788c-7879-404a-a75d-661bedba5afb.png)

For more information: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description